src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -913,9 +912,10 @@ MStatus PRTModifierAction::addParameter(MFnDependencyNode& node, MObject& attr,
 		MObject attrUserSet = nAttrUserSet.create(tAttr.name() + ATTRIBUTE_USER_SET_SUFFIX,
 		                                          tAttr.shortName() + ATTRIBUTE_USER_SET_SUFFIX,
 		                                          MFnNumericData::kBoolean, false, &stat);
+		if (stat != MS::kSuccess)
+			throw stat;

```

Comment:
why not just return the non-success status?

Reply:
now returning status